python/tvm/relay/frontend/tflite.py

```diff
@@ -2726,7 +2726,13 @@ def convert_quantize(self, op):
         assert len(input_tensors) == 1, "input tensors length should be 1"
         input_tensor = input_tensors[0]
         input_tensor_type_str = self.get_tensor_type_str(input_tensor.tensor.Type())
-        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        if self.has_expr(input_tensor.tensor_idx):
```

Comment:
Does it make sense to use `self.get_tensor_expr` here?

Reply:
Replaced